x-pack/legacy/plugins/siem/public/apps/start_app.tsx

```diff
@@ -17,45 +18,67 @@ import { BehaviorSubject } from 'rxjs';
 import { pluck } from 'rxjs/operators';
 import { I18nContext } from 'ui/i18n';
 
+import { DEFAULT_DARK_MODE } from '../../common/constants';
 import { ErrorToastDispatcher } from '../components/error_toast_dispatcher';
-import { KibanaConfigContext } from '../lib/adapters/framework/kibana_framework_adapter';
+import { compose } from '../lib/compose/kibana_compose';
 import { AppFrontendLibs } from '../lib/lib';
 import { PageRouter } from '../routes';
 import { createStore } from '../store';
 import { GlobalToaster, ManageGlobalToaster } from '../components/toasters';
 import { MlCapabilitiesProvider } from '../components/ml/permissions/ml_capabilities_provider';
+import { useKibanaUiSetting } from '../lib/settings/use_kibana_ui_setting';
 
-export const startApp = async (libs: AppFrontendLibs) => {
+const startApp = (libs: AppFrontendLibs) => {
   const history = createHashHistory();
 
   const libs$ = new BehaviorSubject(libs);
 
   const store = createStore(undefined, libs$.pipe(pluck('apolloClient')));
 
-  libs.framework.render(
-    <EuiErrorBoundary>
-      <I18nContext>
-        <ManageGlobalToaster>
-          <ReduxStoreProvider store={store}>
-            <ApolloProvider client={libs.apolloClient}>
-              <ThemeProvider
-                theme={() => ({
-                  eui: libs.framework.darkMode ? euiDarkVars : euiLightVars,
-                  darkMode: libs.framework.darkMode,
-                })}
-              >
-                <KibanaConfigContext.Provider value={libs.framework}>
+  const AppPluginRoot = memo(() => {
```

Comment:
Should we add?

``` ts
AppPluginRoot.displayName = 'AppPluginRoot';
```

![image](https://user-images.githubusercontent.com/2946766/63182412-bd087000-c00f-11e9-9dc8-9f068167707e.png)

Reply:
oh yeah, I forget about that trick